python/ray/util/spark/utils.py

```diff
@@ -383,18 +382,17 @@ def acquire_lock(file):
         mode = os.O_RDWR | os.O_CREAT | os.O_TRUNC
         file_ref = os.open(file, mode)
         locked_file_ref = None
-        lock_time = 10.0
-        start = current = time.time()
-        while current < start + lock_time:
+        # Allow for retrying getting a file lock a maximum number of seconds
+        max_lock_iter = 60 * 2
```

Comment:
Let's increase the timeout to be 600 seconds, which is safer (Considering some instance has large cpu cores and might launch many ray workers)